lib/puppet/gettext/config.rb

```diff
@@ -5,6 +5,14 @@ module Puppet::GettextConfig
   POSIX_PATH = File.absolute_path('../../../../../share/locale', File.dirname(__FILE__))
   WINDOWS_PATH = File.absolute_path('../../../../../../../puppet/share/locale', File.dirname(__FILE__))
 
+  begin
+    require 'gettext-setup'
```

Comment:
We do this instead of use a feature because we initialize gettext before features are loaded.